pkg/kapp/clusterapply/delete_change.go

```diff
@@ -56,6 +56,11 @@ func (c DeleteChange) IsDoneApplying() (ctlresm.DoneApplyState, []string, error)
 		return ctlresm.DoneApplyState{Done: true, Successful: true, Message: "Resource orphaned"}, nil, nil
 	}
 
+	if res.IsDeleting() && len(res.Finalizers()) > 0 {
```

Comment:
Can we add some test case for it?

Reply:
Rohit looks like we don't store progress log somewhere. Until the finalizer is not removed the progress log get stuck in between and once we apply a patch to remove finalizer then I think there is no way to get the progress log once the resource deletion is complete.